### PR TITLE
Feature/trsv2449 update v1 public dashboard to cater new invitation journey statues

### PR DESCRIPTION
**Pull request by osimuka**

### CODE REVIEW

File: trade_remedies_public/templates/dashboard.html [hunk 1]
```diff
@@ -136,7 +154,7 @@ <h1 class="heading-medium">Your 3rd Party invites</h1>
                                 made, as well as being able to request a reconsideration of a final determination.</p>
                         </li>
                         {% if pre_manage_team and is_org_owner %}
-                            <li><a class="link bold" href="{% url 'manage-users' %}">Manage your team</a>
+                            <li><a class="link bold" href="{% url 'manage-users' %}">Manage users</a>
```
Comment (nahmed99) [inline trade_remedies_public/templates/dashboard.html:157]: This is before the user has done anything (after first registration). On ticket TRSv2-449 (comments), Nick stated that there is no need to change this.

File: trade_remedies_public/core/views.py [hunk 2]
```diff
@@ -413,11 +414,47 @@ def get(self, request, *args, **kwargs):
...
 
+        pending_invitation_count = 0
+        pending_invitation_deficient_docs_count = 0
+        invitation_waiting_trs_approval_count = 0
+
         # Let's get the cases where the user is awaiting approval
         invitations = v2_client.invitations(
-            contact_id=request.user.contact["id"],
+            organisation_id=self.request.user.contact["organisation"]["id"],
             invitation_type=2,
```
Comment (chris-pettinga) [inline trade_remedies_public/core/views.py:424]: we need to get both own-org and representative invites but NOT caseworker invites. SO basically any invitation that is invitation_type either 1 or 2

File: trade_remedies_public/core/views.py [hunk 3]
```diff
@@ -396,6 +396,7 @@ def get(self, request, *args, **kwargs):
         invite_submissions = []
         case_to_roi = {}
         v2_client = TRSAPIClient(token=request.user.token)
+
         if organisation:
```
Comment (chris-pettinga) [inline trade_remedies_public/core/views.py:400]: we should check first that the current logged-in user is an admin of their organisation before we bother to do this.

File: trade_remedies_public/templates/dashboard.html [hunk 4]
```diff
@@ -92,30 +92,44 @@ <h1 class="heading-medium">Your draft registrations of interest</h1>
...
-                                                    {{ invite.case.reference }}
-                                                    - {% format_date invite.created_at '%d %b %Y' %}
-                                                </a>
-                                            {% endif %}
-                                            ({{ invite.name }} - {{ invite.status.name }})
-                                        </LI>
-                                    {% endif %}
-                                {% endfor %}
-                            </UL>
+                        {% if is_org_owner or pending_invitation_count or pending_invitation_deficient_docs_count or pending_invitation_waiting_trs_approval_count %}
```
Comment (chris-pettinga) [inline trade_remedies_public/templates/dashboard.html:95]: org_owner needs to be always True, so 
```suggestion
                        {% if is_org_owner and pending_invitation_count or pending_invitation_deficient_docs_count or pending_invitation_waiting_trs_approval_count %}
```

**Review by chris-pettinga — CHANGES_REQUESTED** [review]

**Pull request merged**